Mathlib/Analysis/Normed/Module/Complemented.lean

```diff
@@ -128,4 +128,119 @@ theorem ClosedComplemented.of_quotient_finiteDimensional [CompleteSpace 𝕜]
   haveI : FiniteDimensional 𝕜 q := (p.quotientEquivOfIsCompl q hq).finiteDimensional
   exact .of_isCompl_isClosed hq hp q.closed_of_finiteDimensional
 
+variable (p q)
+
+/-- Idempotent corresponding to a complemented subspace. -/
+def idempotentOfClosedCompl (h : IsCompl p q) (hp : IsClosed (p : Set E))
+    (hq : IsClosed (q : Set E)) : E →L[𝕜] E :=
```

Comment:
This is the composition of `Submodule.subtypeL` and `linearProjOfClosedCompl`.

Reply:
Done, thanks!